habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -624,6 +656,63 @@ class NavToObjSuccessMeasurementConfig(MeasurementConfig):
     success_angle_dist: float = 0.261799
 
 
+@attr.s(auto_attribs=True, slots=True)
+class DistToPickGoalMeasurementConfig(DistToGoalMeasurementConfig):
+    type: str = "DistToPickGoal"
+
+
+@attr.s(auto_attribs=True, slots=True)
+class DistToPlaceGoalMeasurementConfig(DistToGoalMeasurementConfig):
+    type: str = "DistToPlaceGoal"
+
+
+@attr.s(auto_attribs=True, slots=True)
+class CatNavRotDistToGoalMeasurementConfig(RotDistToGoalMeasurementConfig):
```

Comment:
You have only added `Cat` in this. Is that a mistake or there is some reason?

Reply:
Will use the prefix 'Ovmm' for all these sensors

Reply:
No reason. I think just different people adding stuff for different things. CatNav was added for skill-wise nav and others were added for integrated metrics

Reply:
Updated